ansible/roles/augeas/tasks/main.yml

```diff
@@ -99,3 +99,17 @@
     - packages
     - augeas
 
+- name: install augeas (Ubuntu 24.04)
+  apt: 
+    pkg: 
+    - libaugeas0
+    - augeas-lenses
+    - augeas-tools
+    - python3-augeas
+    - python3-libxml2
+    - libpython3.10
```

Comment:
```
E: Unable to locate package libpython3.10
```

Reply:
Thanks, fixed in https://github.com/AtlasOfLivingAustralia/ala-install/pull/833/commits/e1d1b37b2643ab22458a72c18ad50a0de50851e3